packages/integrations/image/src/build/ssg.ts

```diff
@@ -67,6 +118,9 @@ export async function ssgBuild({ loader, staticImages, config, outDir, logLevel
 		let inputFile: string | undefined = undefined;
 		let inputBuffer: Buffer | undefined = undefined;
 
+		// tracks the cache duration for the original source image
+		let expires: number = 0;
```

Comment:
eslint doesn't like you typing this :/

Reply:
Yep, kind of an aggressive tslint rule there 🤷